mmc-google-analytics/src/test/unit/java/info/novatec/metricscollector/google/RequestBuilderTest.java

```diff
@@ -0,0 +1,188 @@
+package info.novatec.metricscollector.google;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.google.api.services.analyticsreporting.v4.AnalyticsReporting;
+import com.google.api.services.analyticsreporting.v4.model.Dimension;
+import com.google.api.services.analyticsreporting.v4.model.DimensionFilter;
+import com.google.api.services.analyticsreporting.v4.model.DimensionFilterClause;
+import com.google.api.services.analyticsreporting.v4.model.Metric;
+import com.google.api.services.analyticsreporting.v4.model.ReportRequest;
+
+import info.novatec.metricscollector.google.exception.IORuntimeException;
+
+
+/**
+ * Created by DBE on 15.09.2017.
+ */
+public class RequestBuilderTest {
+
+    RequestBuilder requestBuilder;
+    AnalyticsReporting analyticsReporting;
+
+    @Before
+    public void init() {
+        analyticsReporting = mock(AnalyticsReporting.class);
+        requestBuilder = new RequestBuilder(analyticsReporting).prepareRequest();
+
+    }
+
+    @Test
+    public void prepareFirstRequest() {
```

Comment:
From naming - a better option would be: assertInitialStateOfRequestBuilderIsValid().